PokeCharts/Client/PokeAPIClient.cs

```diff
@@ -0,0 +1,71 @@
+using System.Net.Http;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Client
+{
+    public class PokeAPIClient
+    {
+        private readonly HttpClient _httpClient;
+        private string _urlApi = "https://beta.pokeapi.co/graphql/v1beta";
+
+        public PokeAPIClient()
+        {
+            _httpClient = new HttpClient();
```

Comment:
Should handled with the singleton collection, [see here](https://github.com/yannickcpnv/TestingWebApi/blob/1591fe945d901afdf5274f49ef64036b95583bec/TestingWebApi.Core/Startup.cs#L21)